{"ip":"::1","level":"error","message":"Rate limit exceeded:","path":"/health","service":"acquisitions-api","userAgent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/140.0.0.0 Safari/537.36"}
{"ip":"::1","level":"error","message":"Rate limit exceeded:","path":"/health","service":"acquisitions-api","userAgent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/140.0.0.0 Safari/537.36"}
{"ip":"::1","level":"error","message":"Rate limit exceeded:","path":"/health","service":"acquisitions-api","userAgent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/140.0.0.0 Safari/537.36"}
{"ip":"::1","level":"error","message":"Rate limit exceeded:","path":"/health","service":"acquisitions-api","userAgent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/140.0.0.0 Safari/537.36"}
{"ip":"::1","level":"error","message":"Rate limit exceeded:","path":"/health","service":"acquisitions-api","userAgent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/140.0.0.0 Safari/537.36"}
{"ip":"::ffff:172.20.0.1","level":"error","message":"Bot detected:","path":"/api/auth/sign-in","service":"acquisitions-api","userAgent":"PostmanRuntime/7.46.1"}
{"ip":"::ffff:172.20.0.1","level":"error","message":"Bot detected:","path":"/api/auth/sign-out","service":"acquisitions-api","userAgent":"PostmanRuntime/7.46.1"}
{"ip":"::ffff:172.20.0.1","level":"error","message":"Bot detected:","path":"/api/auth/sign-up","service":"acquisitions-api","userAgent":"PostmanRuntime/7.46.1"}
{"ip":"::ffff:172.20.0.1","level":"error","message":"Rate limit exceeded:","path":"/health","service":"acquisitions-api","userAgent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/140.0.0.0 Safari/537.36"}
{"ip":"::1","level":"error","message":"Rate limit exceeded:","path":"/health","service":"acquisitions-api"}
{"ip":"::ffff:172.20.0.1","level":"error","message":"Bot detected:","path":"/api/auth/sign-up","service":"acquisitions-api","userAgent":"PostmanRuntime/7.46.1"}
{"ip":"::ffff:172.20.0.1","level":"error","message":"Bot detected:","path":"/api/auth/sign-up","service":"acquisitions-api","userAgent":"PostmanRuntime/7.46.1"}
{"ip":"::ffff:172.20.0.1","level":"error","message":"Bot detected:","path":"/api/auth/sign-up","service":"acquisitions-api","userAgent":"PostmanRuntime/7.46.1"}
{"ip":"::ffff:172.20.0.1","level":"error","message":"Bot detected:","path":"/api/auth/sign-up","service":"acquisitions-api","userAgent":"PostmanRuntime/7.46.1"}
{"ip":"::ffff:172.20.0.1","level":"error","message":"Bot detected:","path":"/api/auth/sign-up","service":"acquisitions-api","userAgent":"PostmanRuntime/7.46.1"}
{"ip":"::ffff:172.20.0.1","level":"error","message":"Bot detected:","path":"/api/auth/sign-up","service":"acquisitions-api","userAgent":"PostmanRuntime/7.46.1"}
{"ip":"::ffff:172.20.0.1","level":"error","message":"Bot detected:","path":"/api/auth/sign-up","service":"acquisitions-api","userAgent":"PostmanRuntime/7.46.1"}
{"ip":"::ffff:172.20.0.1","level":"error","message":"Bot detected:","path":"/api/auth/sign-up","service":"acquisitions-api","userAgent":"PostmanRuntime/7.46.1"}
{"ip":"::ffff:172.20.0.1","level":"error","message":"Bot detected:","path":"/api/auth/sign-up","service":"acquisitions-api","userAgent":"PostmanRuntime/7.46.1"}
{"level":"error","message":"Error while creating user User with this email already exists","service":"acquisitions-api","stack":"Error: User with this email already exists\n    at createUser (file:///app/src/services/auth.service.js:29:19)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async singup (file:///app/src/controllers/auth.controller.js:19:22)"}
{"level":"error","message":"Error in singup Error while creating user","service":"acquisitions-api","stack":"Error: Error while creating user\n    at createUser (file:///app/src/services/auth.service.js:48:15)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async singup (file:///app/src/controllers/auth.controller.js:19:22)"}
{"ip":"::ffff:172.20.0.1","level":"error","message":"Bot detected:","path":"/api/auth/sign-up","service":"acquisitions-api","userAgent":"PostmanRuntime/7.46.1"}
{"ip":"::ffff:172.20.0.1","level":"error","message":"Bot detected:","path":"/api/auth/sign-in","service":"acquisitions-api","userAgent":"PostmanRuntime/7.46.1"}
{"ip":"::ffff:172.20.0.1","level":"error","message":"Bot detected:","path":"/api/auth/sign-in","service":"acquisitions-api","userAgent":"PostmanRuntime/7.46.1"}
{"ip":"::ffff:172.20.0.1","level":"error","message":"Bot detected:","path":"/api/auth/sign-up","service":"acquisitions-api","userAgent":"PostmanRuntime/7.46.1"}
{"level":"error","message":"Error while creating user User with this email already exists","service":"acquisitions-api","stack":"Error: User with this email already exists\n    at createUser (file:///app/src/services/auth.service.js:29:19)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async singup (file:///app/src/controllers/auth.controller.js:19:22)"}
{"level":"error","message":"Error in singup Error while creating user","service":"acquisitions-api","stack":"Error: Error while creating user\n    at createUser (file:///app/src/services/auth.service.js:48:15)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async singup (file:///app/src/controllers/auth.controller.js:19:22)"}
{"level":"error","message":"Error while creating user User with this email already exists","service":"acquisitions-api","stack":"Error: User with this email already exists\n    at createUser (file:///app/src/services/auth.service.js:29:19)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async singup (file:///app/src/controllers/auth.controller.js:19:22)"}
{"level":"error","message":"Error in singup Error while creating user","service":"acquisitions-api","stack":"Error: Error while creating user\n    at createUser (file:///app/src/services/auth.service.js:48:15)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async singup (file:///app/src/controllers/auth.controller.js:19:22)"}
{"level":"error","message":"Error while creating user User with this email already exists","service":"acquisitions-api","stack":"Error: User with this email already exists\n    at createUser (file:///app/src/services/auth.service.js:29:19)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async singup (file:///app/src/controllers/auth.controller.js:19:22)"}
{"level":"error","message":"Error in singup Error while creating user","service":"acquisitions-api","stack":"Error: Error while creating user\n    at createUser (file:///app/src/services/auth.service.js:48:15)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async singup (file:///app/src/controllers/auth.controller.js:19:22)"}
{"level":"error","message":"Error while creating user User with this email already exists","service":"acquisitions-api","stack":"Error: User with this email already exists\n    at createUser (file:///app/src/services/auth.service.js:29:19)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async singup (file:///app/src/controllers/auth.controller.js:19:22)"}
{"level":"error","message":"Error in singup Error while creating user","service":"acquisitions-api","stack":"Error: Error while creating user\n    at createUser (file:///app/src/services/auth.service.js:48:15)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async singup (file:///app/src/controllers/auth.controller.js:19:22)"}
{"cause":{"name":"NeonDbError","sourceError":{}},"level":"error","message":"Error while creating user Failed query: select \"id\", \"name\", \"email\", \"password\", \"role\", \"created_at\", \"updated_at\" from \"users\" where \"users\".\"email\" = $1 limit $2\nparams: user123@example.com,1","params":["user123@example.com",1],"query":"select \"id\", \"name\", \"email\", \"password\", \"role\", \"created_at\", \"updated_at\" from \"users\" where \"users\".\"email\" = $1 limit $2","service":"acquisitions-api","stack":"Error: Failed query: select \"id\", \"name\", \"email\", \"password\", \"role\", \"created_at\", \"updated_at\" from \"users\" where \"users\".\"email\" = $1 limit $2\nparams: user123@example.com,1\n    at NeonHttpPreparedQuery.queryWithCache (file:///D:/DevOps-HandsOn/Acquisitions/node_modules/drizzle-orm/pg-core/session.js:41:15)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async NeonHttpPreparedQuery.execute (file:///D:/DevOps-HandsOn/Acquisitions/node_modules/drizzle-orm/neon-http/session.js:45:20)\n    at async createUser (file:///D:/DevOps-HandsOn/Acquisitions/src/services/auth.service.js:27:30)\n    at async singup (file:///D:/DevOps-HandsOn/Acquisitions/src/controllers/auth.controller.js:19:22)"}
{"level":"error","message":"Error in singup Error while creating user","service":"acquisitions-api","stack":"Error: Error while creating user\n    at createUser (file:///D:/DevOps-HandsOn/Acquisitions/src/services/auth.service.js:48:15)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async singup (file:///D:/DevOps-HandsOn/Acquisitions/src/controllers/auth.controller.js:19:22)"}
{"ip":"::1","level":"error","message":"Bot detected:","path":"/api/auth/sign-up","service":"acquisitions-api","userAgent":"PostmanRuntime/7.46.1"}
{"ip":"::1","level":"error","message":"Bot detected:","path":"/api/auth/sign-up","service":"acquisitions-api","userAgent":"PostmanRuntime/7.46.1"}
{"ip":"::1","level":"error","message":"Bot detected:","path":"/api/auth/sign-up","service":"acquisitions-api","userAgent":"PostmanRuntime/7.46.1"}
{"ip":"::1","level":"error","message":"Bot detected:","path":"/api/auth/sign-up","service":"acquisitions-api","userAgent":"HTTPie"}
{"cause":{"name":"NeonDbError","sourceError":{}},"level":"error","message":"Error while creating user Failed query: select \"id\", \"name\", \"email\", \"password\", \"role\", \"created_at\", \"updated_at\" from \"users\" where \"users\".\"email\" = $1 limit $2\nparams: user123@example.com,1","params":["user123@example.com",1],"query":"select \"id\", \"name\", \"email\", \"password\", \"role\", \"created_at\", \"updated_at\" from \"users\" where \"users\".\"email\" = $1 limit $2","service":"acquisitions-api","stack":"Error: Failed query: select \"id\", \"name\", \"email\", \"password\", \"role\", \"created_at\", \"updated_at\" from \"users\" where \"users\".\"email\" = $1 limit $2\nparams: user123@example.com,1\n    at NeonHttpPreparedQuery.queryWithCache (file:///D:/DevOps-HandsOn/Acquisitions/node_modules/drizzle-orm/pg-core/session.js:41:15)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async NeonHttpPreparedQuery.execute (file:///D:/DevOps-HandsOn/Acquisitions/node_modules/drizzle-orm/neon-http/session.js:45:20)\n    at async createUser (file:///D:/DevOps-HandsOn/Acquisitions/src/services/auth.service.js:27:30)\n    at async singup (file:///D:/DevOps-HandsOn/Acquisitions/src/controllers/auth.controller.js:19:22)"}
{"level":"error","message":"Error in singup Error while creating user","service":"acquisitions-api","stack":"Error: Error while creating user\n    at createUser (file:///D:/DevOps-HandsOn/Acquisitions/src/services/auth.service.js:48:15)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async singup (file:///D:/DevOps-HandsOn/Acquisitions/src/controllers/auth.controller.js:19:22)"}
{"ip":"::1","level":"error","message":"Bot detected:","path":"/api/auth/sign-up","service":"acquisitions-api","userAgent":"PostmanRuntime/7.46.1"}
{"ip":"::1","level":"error","message":"Bot detected:","path":"/api/auth/sign-up","service":"acquisitions-api","userAgent":"PostmanRuntime/7.46.1"}
{"ip":"::1","level":"error","message":"Bot detected:","path":"/api/auth/sign-up","service":"acquisitions-api","userAgent":"PostmanRuntime/7.46.1"}
{"ip":"::1","level":"error","message":"Bot detected:","path":"/api/auth/sign-up","service":"acquisitions-api","userAgent":"HTTPie"}
{"ip":"::1","level":"error","message":"Bot detected:","path":"/api/auth/sign-up","service":"acquisitions-api","userAgent":"HTTPie"}
{"ip":"::1","level":"error","message":"Bot detected:","path":"/api/auth/sign-up","service":"acquisitions-api","userAgent":"HTTPie"}
{"ip":"::1","level":"error","message":"Bot detected:","path":"/api/auth/sign-up","service":"acquisitions-api","userAgent":"HTTPie"}
{"cause":{"name":"NeonDbError","sourceError":{}},"level":"error","message":"Error while creating user Failed query: select \"id\", \"name\", \"email\", \"password\", \"role\", \"created_at\", \"updated_at\" from \"users\" where \"users\".\"email\" = $1 limit $2\nparams: user123@example.com,1","params":["user123@example.com",1],"query":"select \"id\", \"name\", \"email\", \"password\", \"role\", \"created_at\", \"updated_at\" from \"users\" where \"users\".\"email\" = $1 limit $2","service":"acquisitions-api","stack":"Error: Failed query: select \"id\", \"name\", \"email\", \"password\", \"role\", \"created_at\", \"updated_at\" from \"users\" where \"users\".\"email\" = $1 limit $2\nparams: user123@example.com,1\n    at NeonHttpPreparedQuery.queryWithCache (file:///D:/DevOps-HandsOn/Acquisitions/node_modules/drizzle-orm/pg-core/session.js:41:15)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async NeonHttpPreparedQuery.execute (file:///D:/DevOps-HandsOn/Acquisitions/node_modules/drizzle-orm/neon-http/session.js:45:20)\n    at async createUser (file:///D:/DevOps-HandsOn/Acquisitions/src/services/auth.service.js:27:30)\n    at async singup (file:///D:/DevOps-HandsOn/Acquisitions/src/controllers/auth.controller.js:19:22)"}
{"level":"error","message":"Error in singup Error while creating user","service":"acquisitions-api","stack":"Error: Error while creating user\n    at createUser (file:///D:/DevOps-HandsOn/Acquisitions/src/services/auth.service.js:48:15)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async singup (file:///D:/DevOps-HandsOn/Acquisitions/src/controllers/auth.controller.js:19:22)"}
{"ip":"::1","level":"error","message":"Bot detected:","path":"/api/auth/sign-up","service":"acquisitions-api","userAgent":"PostmanRuntime/7.46.1"}
{"ip":"::1","level":"error","message":"Bot detected:","path":"/api/auth/sign-up","service":"acquisitions-api","userAgent":"PostmanRuntime/7.46.1"}
{"cause":{"name":"NeonDbError","sourceError":{}},"level":"error","message":"Error while creating user Failed query: select \"id\", \"name\", \"email\", \"password\", \"role\", \"created_at\", \"updated_at\" from \"users\" where \"users\".\"email\" = $1 limit $2\nparams: user123@example.com,1","params":["user123@example.com",1],"query":"select \"id\", \"name\", \"email\", \"password\", \"role\", \"created_at\", \"updated_at\" from \"users\" where \"users\".\"email\" = $1 limit $2","service":"acquisitions-api","stack":"Error: Failed query: select \"id\", \"name\", \"email\", \"password\", \"role\", \"created_at\", \"updated_at\" from \"users\" where \"users\".\"email\" = $1 limit $2\nparams: user123@example.com,1\n    at NeonHttpPreparedQuery.queryWithCache (file:///D:/DevOps-HandsOn/Acquisitions/node_modules/drizzle-orm/pg-core/session.js:41:15)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async NeonHttpPreparedQuery.execute (file:///D:/DevOps-HandsOn/Acquisitions/node_modules/drizzle-orm/neon-http/session.js:45:20)\n    at async createUser (file:///D:/DevOps-HandsOn/Acquisitions/src/services/auth.service.js:27:30)\n    at async singup (file:///D:/DevOps-HandsOn/Acquisitions/src/controllers/auth.controller.js:19:22)"}
{"level":"error","message":"Error in singup Error while creating user","service":"acquisitions-api","stack":"Error: Error while creating user\n    at createUser (file:///D:/DevOps-HandsOn/Acquisitions/src/services/auth.service.js:48:15)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async singup (file:///D:/DevOps-HandsOn/Acquisitions/src/controllers/auth.controller.js:19:22)"}
{"level":"error","message":"Error while creating user User with this email already exists","service":"acquisitions-api","stack":"Error: User with this email already exists\n    at createUser (file:///app/src/services/auth.service.js:29:19)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async singup (file:///app/src/controllers/auth.controller.js:19:22)"}
{"level":"error","message":"Error in singup Error while creating user","service":"acquisitions-api","stack":"Error: Error while creating user\n    at createUser (file:///app/src/services/auth.service.js:48:15)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async singup (file:///app/src/controllers/auth.controller.js:19:22)"}
{"level":"error","message":"Error fetching all users Maximum call stack size exceeded","service":"acquisitions-api","stack":"RangeError: Maximum call stack size exceeded\n    at Array.values (<anonymous>)\n    at file:///app/node_modules/drizzle-orm/utils.js:53:40\n    at Array.reduce (<anonymous>)\n    at orderSelectedFields (file:///app/node_modules/drizzle-orm/utils.js:53:33)\n    at file:///app/node_modules/drizzle-orm/utils.js:63:22\n    at Array.reduce (<anonymous>)\n    at orderSelectedFields (file:///app/node_modules/drizzle-orm/utils.js:53:33)\n    at file:///app/node_modules/drizzle-orm/utils.js:63:22\n    at Array.reduce (<anonymous>)\n    at orderSelectedFields (file:///app/node_modules/drizzle-orm/utils.js:53:33)"}
{"level":"error","message":"Error fetching all users Error fetching all users","service":"acquisitions-api","stack":"Error: Error fetching all users\n    at getAllUsers (file:///app/src/services/user.service.js:16:15)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async getAllUsersController (file:///app/src/controllers/users.controller.js:7:26)"}
{"level":"error","message":"Error fetching all users Maximum call stack size exceeded","service":"acquisitions-api","stack":"RangeError: Maximum call stack size exceeded\n    at Array.values (<anonymous>)\n    at file:///app/node_modules/drizzle-orm/utils.js:53:40\n    at Array.reduce (<anonymous>)\n    at orderSelectedFields (file:///app/node_modules/drizzle-orm/utils.js:53:33)\n    at file:///app/node_modules/drizzle-orm/utils.js:63:22\n    at Array.reduce (<anonymous>)\n    at orderSelectedFields (file:///app/node_modules/drizzle-orm/utils.js:53:33)\n    at file:///app/node_modules/drizzle-orm/utils.js:63:22\n    at Array.reduce (<anonymous>)\n    at orderSelectedFields (file:///app/node_modules/drizzle-orm/utils.js:53:33)"}
{"level":"error","message":"Error fetching all users Error fetching all users","service":"acquisitions-api","stack":"Error: Error fetching all users\n    at getAllUsers (file:///app/src/services/user.service.js:16:15)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async getAllUsersController (file:///app/src/controllers/users.controller.js:7:26)"}
{"level":"error","message":"Error while creating user User with this email already exists","service":"acquisitions-api","stack":"Error: User with this email already exists\n    at createUser (file:///app/src/services/auth.service.js:29:19)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async singup (file:///app/src/controllers/auth.controller.js:19:22)"}
{"level":"error","message":"Error in singup Error while creating user","service":"acquisitions-api","stack":"Error: Error while creating user\n    at createUser (file:///app/src/services/auth.service.js:48:15)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async singup (file:///app/src/controllers/auth.controller.js:19:22)"}
{"level":"error","message":"Error while creating user User with this email already exists","service":"acquisitions-api","stack":"Error: User with this email already exists\n    at createUser (file:///app/src/services/auth.service.js:29:19)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async singup (file:///app/src/controllers/auth.controller.js:19:22)"}
{"level":"error","message":"Error in singup Error while creating user","service":"acquisitions-api","stack":"Error: Error while creating user\n    at createUser (file:///app/src/services/auth.service.js:48:15)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async singup (file:///app/src/controllers/auth.controller.js:19:22)"}
{"level":"error","message":"Error while creating user User with this email already exists","service":"acquisitions-api","stack":"Error: User with this email already exists\n    at createUser (file:///app/src/services/auth.service.js:29:19)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async singup (file:///app/src/controllers/auth.controller.js:19:22)"}
{"level":"error","message":"Error in singup Error while creating user","service":"acquisitions-api","stack":"Error: Error while creating user\n    at createUser (file:///app/src/services/auth.service.js:48:15)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async singup (file:///app/src/controllers/auth.controller.js:19:22)"}
